timbermill-local/src/main/java/com/datorama/oss/timbermill/cron/PersistentFetchJob.java

```diff
@@ -0,0 +1,25 @@
+package com.datorama.oss.timbermill.cron;
+
+import org.quartz.Job;
+import org.quartz.JobExecutionContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.datorama.oss.timbermill.ElasticsearchClient;
+import com.datorama.oss.timbermill.common.ElasticsearchUtil;
+
+public class PersistentFetchJob implements Job {
+
+	private static final Logger LOG = LoggerFactory.getLogger(PersistentFetchJob.class);
+	private static boolean keepRunning = false;
+
+	@Override public void execute(JobExecutionContext context) {
+		if (!keepRunning) {
```

Comment:
keepRuning will stop jobs to be executed while another job os successfully sending bulks to es. not sure if we need a lock here?

Reply:
OK but let's change it a bit to make it more understandable:
* change keepRunning name to currentlyRunning
* set it to false once you finish running
* retryFailedRequestsFromDisk souldn't be assigned, just keep running until the return value is false.